testplan/docs/director/openebs/directorOpenebs.md

```diff
@@ -37,33 +37,107 @@ This is supported only for OpenEBS installation done via DOP.
 
 This can be perfromed only post Control plane components are upgraded. Data plane upgrade is supported for any mode of OpenEBS installation i.e OpenEBS dataplane can be upgraded for non DOP installed case as well.
 
-| TCID | Breif Description | #Issue |
+| TCID | Brief Description | #Issue |
 | ---- | ----------------- | ------ |
 |      |                   |        |
 |      |                   |        |
 |      |                   |        |
 
 ###  Pool Provisioning
 - #### cStor Pool Provisioning 
-
-
-| TCID | Breif Description | #Issue |
-| ---- | ----------------- | ------ |
-|      |                   |        |
-|      |                   |        |
-|      |                   |        |
+  - [ ] TCID-CSP-REC-001
+    - **UseCase**: Director should **list all** cstor pool recommendations
```

Comment:
Strategy/Summary page having description should be good, IMO, Details, ID, etc can go into actual TC. else page becomes cluttered.

Reply:
This was the way to put just theory & no code. Just information without syntaxes.
User should be able to come here understand & add their cases.